
string destination
---
bool success
string info
